resharper/resharper-unity/src/Rider/CodeInsights/RiderUnityImplicitUsageHighlightingContributor.cs

```diff
@@ -97,27 +115,39 @@ public override IEnumerable<BulbMenuItem> CreateAdditionalMenuItem(IDeclaration
                 };
             }
 
+
             return EmptyList<BulbMenuItem>.Instance;
         }
 
         internal class UnityFindUsagesNavigationAction : BulbActionBase
         {
             private readonly IDeclaredElement myDeclaredElement;
             private readonly UnityEditorFindRequestCreator myCreator;
+            [NotNull] private readonly ITooltipManager myTooltipManager;
+            [NotNull] private readonly ConnectionTracker myTracker;
 
-            public UnityFindUsagesNavigationAction([NotNull]IDeclaredElement method, [NotNull]UnityEditorFindRequestCreator creator)
+            public UnityFindUsagesNavigationAction([NotNull]IDeclaredElement method, [NotNull]UnityEditorFindRequestCreator creator,[NotNull] ITooltipManager tooltipManager, [NotNull] ConnectionTracker tracker)
             {
                 myDeclaredElement = method;
                 myCreator = creator;
+                myTooltipManager = tooltipManager;
+                myTracker = tracker;
             }
             
             protected override Action<ITextControl> ExecutePsiTransaction(ISolution solution, IProgressIndicator progress)
             {
+                if (myTracker.LastCheckResult == UnityEditorState.Disconnected)
```

Comment:
Maybe not accurate. As I remember some other states are also disconnected, consider the following
if (result.Result != UnityEditorState.Refresh && result.Result != UnityEditorState.Disconnected)